src/app/tests/suites/certification/Test_TC_CC_9_1.yaml

```diff
@@ -432,7 +615,102 @@ tests:
           constraints:
               type: int16u
               minValue: ColorLoopStartEnhancedHueStep8d
-              maxValue: 65535
+              maxValue: enhancedCurrentHueMaxValue
+
+    - label: "Wait for 5s"
+      PICS: CC.S.F02 && CC.S.F01
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 5000
+
+    - label: "Read EnhancedCurrentHue attribute from DUT"
+      command: "readAttribute"
+      attribute: "EnhancedCurrentHue"
+      PICS: CC.S.F02 && CC.S.F01 && CC.S.A4000 && CC.S.C44.Rsp
```

Comment:
The pics here look wrong. 
A4000 is implied by F01 (EHUE) becasue it's mandatory, but why is a CT command in here?